### ISSUE TITLE
No information on how to turn on Microsoft 365 defender. This should be the first item.

### ISSUE THREAD

**Issue by Sturmey66** [MicrosoftDocs/microsoft-365-docs#12809]
[Page title is: Turn on Microsoft 365 Defender. 
Nothing on this page links to where in my account I can get to the page that allows me to turn on Defender. most of the page talks about license and eligibility, but never gives information about how to get to the switch to turn it on. Makes the article worthless.]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9a7f625c-f3ef-882d-231f-6fef18e14137
* Version Independent ID: 3f335d54-6d65-d49a-e517-2c3a9a95e44d
* Content: [Turn on Microsoft 365 Defender](https://learn.microsoft.com/en-us/microsoft-365/security/defender/m365d-enable?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender/m365d-enable.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender/m365d-enable.md)
* Service: **microsoft-365-security**
* Sub-service: **m365d**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by DMoenks** [comment]
In order, the referenced article states "_Microsoft 365 Defender automatically turns on when eligible customers with the required permissions visit Microsoft 365 Defender portal_", "_A license to a Microsoft 365 security product generally entitles you to use Microsoft 365 Defender without additional licensing cost_", "_You must be one of the following roles to turn on Microsoft 365 Defender: [...]_" and "_Onboarding to Microsoft 365 Defender is simple. From the navigation menu, select any item [...] to initiate the onboarding process_"
So basically, when a Defender product is licensed, the first time a member of one of the listed admistrative roles accesses Defender portal, it provisions itself

**Comment by msbemba** [comment]
@Sturmey66 Thank you for your feedback. As mentioned in the article Microsoft 365 Defender automatically turns on when eligible customers with the required permissions visit the Microsoft 365 Defender portal. Read this article to understand various prerequisites and how Microsoft 365 Defender is provisioned.

Hope this helps!

Thanks
Sri

**Comment by yogkumgit** [comment]
@msbemba Thank you very much for the contribution and sharing this explanation. @Sturmey66 Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by Sturmey66** [comment]
Nothing in this interaction has resolved this issue. Your documentation is still lacking and the answer I received shows how little you know about how to answer questions. This is exactly why people hate Windows. You gave me a useless answer that doesn't address what I asked, and somehow I'm supposed to be happy about it.

**Comment by yogkumgit** [comment]
@msbemba Please review and update. Thanks

**Comment by msbemba** [comment]
@yogkumgit  There is no setting as such, Microsoft 365 Defender automatically turns on when eligible customers with the required permissions visit the Microsoft 365 Defender portal.

 We can add author to provide more information if there are any.

Thanks
Sri
